### PR TITLE
EC 238 Display sync archive queue on receiver dashboard and tab listing queue items 

### PR DESCRIPTION
**Pull request by danielchirinda**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { SyncArchiveLoaded } from './state/sync-archive.actions';
+import { GET_SYNC_ARCHIVE } from './state/sync-archive.reducer';
+
+@Component({
+  selector: 'app-archive',
+  templateUrl: './archive.component.html',
+  styleUrls: ['./archive.component.scss']
+})
+export class ArchiveComponent extends BaseListingComponent implements OnInit {
+
```
Comment (wluyima) [inline webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:17]: Please always format your code to ensure you have no extra lines like here, the IDE has this feature built-in to format code

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+				this.events = countAndItems?.items;
+				this.reRender();
+			}
+		);
+
+		this.loadDetails();
+
+	}
+
+	loadDetails(): void {
```
Comment (wluyima) [inline webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:47]: Copy and paste? This should be something like `loadArchives()`

**Review by wluyima — COMMENTED** [review]
Will this also have the search feature?

**Comment by wluyima** [comment]
This looks good in general, so what does it depend on from the PR for EC-237?

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts (hunk 1 above)
Comment (wluyima) [reply webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:17]: ?

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts (hunk 2 above)
Comment (wluyima) [reply webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:47]: Did you see this comment?

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts (hunk 1 above)
Comment (wluyima) [reply webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:17]: And this comment?

File: webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts (hunk 2 above)
Comment (danielchirinda) [reply webapp/src/main/webapp/src/app/receiver/archive/archive.component.ts:47]: Yes, i will work on them today.

**Comment by wluyima** [comment]
I'm confused, please don't mix the changes for this PR with those of EC-237, lets wait to merge the PR for EC-237 that way this PR only contains changes for EC-238

**Comment by wluyima** [comment]
Please don’t mix the changes in the branch for EC-237 with EC-238 that way this PR only contains changes for EC-238, when we merge the PR for EC-237 you will then update your EC-238 branch and the PR will be clean

**Pull request closed without merging**